join-domain/elx/sssd/files/join.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -eu
```

Comment:
Does this still work if we also add pipefail?

```suggestion
set -eu -o pipefail
```

Reply:
I'd had it with `-o pipefail` early on (and was starting this `join.sh` as a copy of PBIS's), but may have lost it when I decided to start from scratch.